src/test/suite/extension.test.ts

```diff
@@ -1,15 +1,58 @@
 import * as assert from 'assert';
 
-// You can import and use all API from the 'vscode' module
-// as well as import your extension to test it
 import * as vscode from 'vscode';
-// import * as myExtension from '../extension';
+
+import { registerCommands, launchMongoShell } from '../../commands';
+
+import { TestExtensionContext } from './stubs';
 
 suite('Extension Test Suite', () => {
-	vscode.window.showInformationMessage('Start all tests.');
+	vscode.window.showInformationMessage('Starting tests...');
+
```

Comment:
> [TABS]

Reminds me. Did I check in `.vscode/settings.json` to auto-format on save?

Reply:
Yup - added rules for indent there and the tslint config. Updated any files that were using 4 space indent 🤮or tabs.